assets/src/edit-story/components/canvas/useCanvasSelectionCopyPaste.js

```diff
@@ -117,8 +121,16 @@ function useCanvasSelectionCopyPaste(container) {
         if (html) {
           const template = document.createElement('template');
           template.innerHTML = html;
+          let copyingStoryElement = false;
```

Comment:
I think structure here should be a little different. It should basically be a prioritized list of matchers that try recognize and process the posted content. And the first that matches would win.

I.e.  the code should be:
 1. Try to process special stories comment. If succeeds, exit.
 2. Try to process x. If succeeds, exit.
 3. Etc.
 4. Try to process it as a text. If succeeds, exit.

In other words this is a kind of like a `for () { if (processed) break}`. But since we only have two matchers for now, we can arrange code whichever way looks expedient, as long as "tries" are separated clearly.